tests/asahirov/test_hw7_fibo.py

```diff
@@ -0,0 +1,43 @@
+"""
+Create decorator which would remember answers,
+and if answer was given, do not make a call to a function
+"""
+
+
+def remember_answer(func):
+    answer = {}
+    call_count = {"count": 0}
+
+    def remember(*args):
+        try:
+            hash(args)
+        except Exception as e:
+            raise TypeError("input args not hashable ")
+        result = func(*args)
+        if result not in answer:
+            call_count["count"] += 1
+            answer[result] = result
+        return answer[result]
+
+    remember.call_count = call_count
+    return remember
```

Comment:
we write tests for function but not function for tests, it s not the approach you should go forward